tsfile/src/main/java/org/apache/iotdb/tsfile/write/writer/TsFileIOWriter.java

```diff
@@ -69,14 +67,19 @@
   }
 
   protected TsFileOutput out;
-  protected List<ChunkGroupMetaData> chunkGroupMetaDataList = new ArrayList<>();
   protected boolean canWrite = true;
   protected int totalChunkNum = 0;
   protected int invalidChunkNum;
   protected File file;
-  private ChunkGroupMetaData currentChunkGroupMetaData;
+  protected List<List<ChunkMetaData>> chunkGroupMetaDataList = new ArrayList<>();
+  protected List<String> deviceList = new ArrayList<>();
+  protected List<ChunkMetaData> chunkMetaDataList = new ArrayList<>();
+  private static Map<Path, List<ChunkMetaData>> timeseriesMetadataMap = new TreeMap<>();
   private ChunkMetaData currentChunkMetaData;
   private long markedPosition;
+  private static Map<String, int[]> deviceOffsetsMap = new HashMap<>();
```

Comment:
Please remove `static`. The reason is the same as above.

Reply:
Fixed.